integration_tests/testsuite/driver.py

```diff
@@ -67,8 +67,17 @@ def _main():
     parser.add_argument('--yaml', '-y',
                         help='optional path to app.yaml file',
                         required=False)
+    parser.add_argument('--environment', '-e',
+                        help='environment to deploy sample application '
```

Comment:
As per https://docs.python.org/3/library/argparse.html#choices, you can add `choices=test_util.ENVS` here to validate the flag, and get rid of lines 77-80 below.